vue.config.js

```diff
@@ -82,9 +88,6 @@ module.exports = {
       manifestTransformer: (manifest) => {
         manifest.content_security_policy =
           "script-src 'self' 'unsafe-eval' 'unsafe-inline' https://cdn.segment.com 'sha256-ZgDy59Dh4jH9g/vcPYFpoQ1wumB4IdPEOS1BJc08i+Y='; object-src 'self';"
-        manifest.externally_connectable = {
```

Comment:
why was it removed? it won't affect anytrhing?

Reply:
yes in this case we are not using anymore the ledger bridge so we don't need this settings